omegaconf/omegaconf.py

```diff
@@ -508,7 +531,9 @@ def resolver_wrapper(
     @staticmethod
     def get_resolver(
         name: str,
```

Comment:
This should probably not be a public method.
it's exposing internal details.

Reply:
Feel free to change it (this predates #445)

Reply:
It can be done in the pass to improve the resolver API (#370).
I think the intended use here is to check if a resolver exists.

Reply:
So is there a need to keep a public API to check the existence of a resolver?

Reply:
We can deprecate this one and ad a proper function to check that.